.github/workflows/test.yml

```diff
@@ -10,22 +10,25 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
       - name: Install Rust stable
-        uses: dtolnay/rust-toolchain@stable
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: stable
+          components: rustfmt
       - name: Run formatting checks
-        run: cargo fmt --check
+        run: cargo +stable fmt --check
```

Comment:
`rust-toolchain` sets the requested toolchain as default. No need to force it with `+stable`. Same for the other jobs.

Reply:
I saw https://github.com/dtolnay/rust-toolchain#outputs and thought it had to be specified, I'll go ahead and move that over and just make sure the CI runs fine still.